src/haz3lcore/statics/Term.re

```diff
@@ -74,10 +80,13 @@ module UTyp = {
     | String
     | Bool => "Base Type"
     | Var => "Type Variable"
+    | Tag => "Sum Constructor"
```

Comment:
since we call it a "Constructor" in user-facing code, might be worth renaming `Tag` to `Constructor` in the code as well?

Reply:
@cyrus- done